modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BaseBatchEngineTaskExecutorTest.java

```diff
@@ -176,7 +176,8 @@ public void createItem(
 				blogPosting.getDatePublished());
 
 			_blogsEntryService.addEntry(
```

Comment:
Once we are going to remove old methods, I have doubts if it is better to include the externalReferenceCode argument in every addEntry signatures and add the "null" value in the usage of the methods instead having here methods with and without externelReferenceCode